largest_sum_contiguous_subarray/LargestSumContiguousSubarray.java

```diff
@@ -14,12 +14,9 @@ public static int largestSumContiguousSubarray(int[] array) {   //  maximum sum
         prevSum = array[0];						// initialize current sum amd previous sum
         currentSum = array[0];
         for (i = 1; i < array.length; i++) {
```

Comment:
I can be declared here only like `for (int i = 1; i < array.length; i++)` and remove line  13